source/index.html.haml

```diff
@@ -75,7 +75,9 @@ hide_metadata: true
           = data.site.name
           News
 
-        = partial :blog_posts, locals: locals
+        -### Load last 3 blog entry summaries ###
+        - page_articles[0..2].reject {|a| !a.published? }.each do |article|
```

Comment:
This rejects unpublished after selecting 3 articles, meaning if you have 1 unpublished, then you'll have 2 articles. If you have 2 unpublished, you'll have 1 article. It should be something like this:

```ruby
- page_articles.select(&:published?).take(3).each do |article|
```